app/teacher

```diff
@@ -0,0 +1,14 @@
+
+class Teacher < Person
+  attr_reader :specialization 
+  attr_writer :specialization 
+
+  def initialize(specialization,id, age, name, parent_permission)
+    super(id, age, name, parent_permission)
+    @specialization  = specialization 
+  end
+
+  def can_use_services?
+    return true
+  end
+end
```

Comment:
- [ ] You should name correctly your file, your file should have an extension `.rb` so it should be `teacher.rb`. Without `.rb` at the end of your file, your code is just like text and not a code. Kindly fix that 🙏 
- [ ] I think the name has a value `Unknown` and the parent_permission should be set up to `true`. You don't need to pass the `id` as an argument 🙃